content/en/docs/tasks/administer-cluster/switch-to-evented-pleg.md

```diff
@@ -5,94 +5,104 @@ content_type: task
 weight: 90
 ---
 
-{{< feature-state for_k8s_version="v1.26" state="beta" >}}
+{{< feature-state for_k8s_version="v1.27" state="beta" >}}
 
 <!-- overview -->
 
-This page shows how to migrate notes to use event based updates for container status. The event-based
+
+This page shows how to migrate nodes to use event based updates for container status. The event-based
 implementation reduces node resource consumption by the kubelet, compared to the legacy approach
 that relies on polling.
 You may know this feature as _evented Pod lifecycle event generator (PLEG)_. That's the name used
 internally within the Kubernetes project for a key implementation detail.
 
+The polling based approach is referred to as _generic PLEG_.
+
 ## {{% heading "prerequisites" %}}
 
 * You need to run a version of Kubernetes that provides this feature.
-  Kubernetes {{< skew currentVersion >}} includes beta support for event-based container
-  status updates. The feature is beta and is disabled by default.
-  {{< version-check >}}
+  Kubernetes v1.27 includes beta support for event-based container
+  status updates. The feature is beta but is _disabled_ by default
+  because it requires support from the container runtime.
+* {{< version-check >}}
   If you are running a different version of Kubernetes, check the documentation for that release.
+* The container runtime in use must support container lifecycle events.
+  The kubelet automatically switches back to the legacy generic PLEG
+  mechanism if the container runtime doesnt not announce support for
+  container lifecycle events, even if you have this feature gate enabled.
 
 <!-- steps -->
 
 ## Why switch to Evented PLEG?
 
-* The current `Generic PLEG` incurs non-negligible overhead due to frequent polling of container statuses.
+* The _Generic PLEG_ incurs non-negligible overhead due to frequent polling of container statuses.
 * This overhead is exacerbated by Kubelet's parallelism, limiting its scalability
```

Comment:
nit: This overhead is exacerbated by the kubelet's parallelism, ...

Not sure what 'the kubelet's pallelism' is referring to

Reply:
Rephrased.